spec/lib/generator_spec.rb

```diff
@@ -8,25 +8,30 @@
       expect(Generator.send(method)).to be_truthy
     end
   end
+  shared_examples 'generates multiple valid' do |method|
+    it method.to_s do
+      expect(5.times { Generator.send(method) }).to be_truthy
```

Comment:
this test should probably also take the model name so we can actually test that the correct number are saved to the database -- `expect(klass.count).to eq(5)`